db/migration/20210329121921_create_shares_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(upCreateSharesTable, downCreateSharesTable)
+}
+
+func upCreateSharesTable(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+create table share
+(
+    id             varchar(255) not null primary key,
```

Comment:
Please fix the indentation

Reply:
That's weird, it looked all fine in VSCode :/
I'll take a look.

Reply:
I guess it was some kind of tab-newline space thing. Fixed it now!